CmrUI/Helpers/GradientHelper.swift

```diff
@@ -29,25 +29,26 @@ class GradientHelper: NSObject {
         gradient.colors = cgcolors
         gradient.startPoint = startPoint
         gradient.endPoint = endPoint
-        view?.layer.addSublayer(gradient)
-        self.setGradientLayer(gradient, for: view)
+        self.layer.addSublayer(gradient)
+        self.setGradientLayer(gradient)
     }
     
-    class func gradientLayer(for view: UIView?) -> CALayer? {
-        if view == nil || view?.layer.sublayers == nil {
+    func gradientLayer() -> CALayer? {
+        if self.layer.sublayers == nil {
             return nil
         }
-        return objc_getAssociatedObject(self, &GradientLayerKey) as? CALayer
+        let ppgog = objc_getAssociatedObject(self, "🔑") as? CALayer
+        return objc_getAssociatedObject(self, "🔑") as? CALayer
```

Comment:
@vmsavka 1. Using associated object here is unnecessary. If you want to identify the layer so as not to add it twice, you can use CALayer's 'name' property. 
2. See my remark on viewDidLoad. If you set the gradient layer on view load, you don't need to check if you need to add it or not every time viewDidAppear is called. 
3. Please use English for keys/values, not emoji.